src/storage.js

```diff
@@ -569,6 +571,18 @@ export class LocalStorageStore {
   getLedgerAppVersion() {
     return this.getItem(LEDGER_APP_VERSION_KEY);
   }
+
+  setMiningServer(server) {
+    this.setItem(MINING_SERVER_KEY, server);
+  }
+
+  resetMiningServer(server) {
```

Comment:
```suggestion
  resetMiningServer() {
```

Reply:
Method no longer exists

Reply:
It's still there

Reply:
Sorry, was looking in the wrong file, fixed